README.md

```diff
@@ -1,10 +1,10 @@
 # Public Curation Protocol
 
 ### High-level overview
-1. CurationFactory.sol allows for easy creation of individual curation contracts
-2. Individual curation contracts are ERC721 collections themselves, with curators recieving a non-transferable "curationReciept" token that contains the information of the listing they have curated. Curators can "remove" listing by burning their curationReciepts
-3. Active listings on a given curation contract can be retreived by the "getListings()" view call on a given Curator.sol proxy, or be using NFT indexers to gather data on all curationReciepts that have been minted from a given curation contract
-4. Listings contain the data specified in the "Listing" struct found in [https://github.com/public-assembly/curation-protocol/blob/main/src/interfaces/ICurator.sol](ICurator.sol)
+1. Factory allows for easy creation of individual curation contracts
+2. Individual curation contracts are ERC721 collections themselves, with curators recieving a non-transferable "listingToken" that contains the information of the listing they have curated. Curators can "remove" listing by burning their curationReciepts
```

Comment:
**Curators** have the ability to call the `addListing()`method from the front-end which mints a non-transferable `listingToken`, bound to their wallet address by `tokenID`, mapping that action to the `Listing` they have curated. 

All `Listing`s are ERC-721s themselves with the following parameters...

Subsequently,`removeListing()` is a method similar to a "burn" method that sends the Curator's `listingToken` to the zero address.

Reply:
I've renamed the `removeListing()` method `burn()` for better compat. with marketplaces.

We can also do `removeListings()` and `burn()` as two diff methods.

Reply:
thats actually interesting idea. feel like it would be nice for ppl to retain proof of their curation even if they havent burned the token

Reply:
mmmm actually I take it back. this reducdes the simplicity of using the indexer to query for active listings, think we should just stick to the burn + burnBatch